apache/yunikorn-web
# [YUNIKORN-2325] Add a chart to display multi-type resource utilisation (Web)

### What is this PR for?
Create a vertical bar chart to display multi-type resource utilisation in YuniKorn UI Nodes page.
- Add 2 componenets,  app-node-utilizations and vertical-bar-chart
- Calculate overall resource usage for each resource type, put top 10 resources in legend  (Order by avg resource usage)
- Only display domaint type's node utilization by default, could display others through legend.


### What type of PR is it?

### Todos
- Remove domaint reosurce chart in dashboard page.

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-2325

### How should this be tested?
- make lint
- make license-check
- yarn test:coverage
- yarn start:srv
- yarn start   (Check the Nodes page)

### Screenshots (if appropriate)
![YUNIKORN-2325 Demo](https://github.com/apache/yunikorn-web/assets/26764036/fbf5486c-0bc0-4c96-934d-16ce9bc4018e)

Recording: https://www.loom.com/share/657bde1a7d834c29b943fbcf70a3ee9a?sid=e6892e8f-c258-4f04-8d3f-1c72435f08c4

### Questions:
NA
